hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/util/LogAdapter.java

```diff
@@ -17,61 +17,40 @@
  */
 package org.apache.hadoop.util;
 
-import org.apache.commons.logging.Log;
 import org.slf4j.Logger;
 
 class LogAdapter {
-  private Log LOG;
-  private Logger LOGGER;
 
-  private LogAdapter(Log LOG) {
-    this.LOG = LOG;
-  }
+  private final Logger LOGGER;
```

Comment:
given its not final, case is wrong. 

also, it's never going to be null, so the checks can be cut down below

Reply:
Done